Sources/SourceKitBazelBSP/RequestHandlers/BuildTargets/TopLevelRuleType.swift

```diff
@@ -18,7 +18,7 @@
 // under the License.
 
 // The list of **top-level rules** we know how to process in the BSP.
-public enum TopLevelRuleType: String, CaseIterable {
+public enum TopLevelRuleType: String, CaseIterable, Sendable {
```

Comment:
Are the sendable changes necessary? Can't see where this would be complained about

Reply:
Ah, that's fine then 👍